content/docs/14-troubleshooting/04-cluster-deployment.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Cluster Deployment"
+metaTitle: "Troubleshooting steps for erros during a cluster deployment"
+metaDescription: "Troubleshooting steps for errors during a cluster deployment."
+icon: ""
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+The following steps will help you troubleshoot errors in the event issues arise while deploying a cluster.  
+
+
+
+## Scenario - Instances Continuously Delete Every 30 Min
+
+An instance is launched and terminated every 30 minutes prior to completion of its deployment, and the **Events Tab** lists errors with the following message:
+
+<br />
+
+```bash
+Failed to update kubeadmControlPlane Connection timeout connecting to Kubernetes Endpoint
+```
+
+This behavior can occur when Kubernetes services for the launched instance fail to start properly. 
+Common reasons for why a service may fail are:
+
+- The specified image could not be pulled from the image repository.
+- The cloud init process failed.
+
+### Debugging Steps
+
+1. Initiate an SSH session with the Kubernetes instance using the SSH key provided during provisioning, and log in as user `spectro`. If you are initiating an SSH session into an installer instance, log in as user `ubuntu`.
+
+    ```shell
+        ssh --identity_file <_pathToYourSSHkey_> spectro@X.X.X.X 
+    ```
+
+2. Elevate the user access. 
+
+    ```shell
+        sudo -i
+    ```
+3. Verify the kubelet service is operational.
+	```shell
+        systemctl status kubelet.service
+    ```
+5. If the kubelet service works as expected, do the following.  
+	- Export the kubeconfig file. 
+
+    ```shell
+        export KUBECONFIG=/etc/kubernetes/admin.conf
+    ```
+	- Connect with the cluster's Kubernetes API. 
+
+    ```shell
+        kubectl get pods --all-namespaces
+    ```
+	- When the connection is established, verify the pods are in a *Running* state. Take note of any pods that are not in *Running* state.
+
+    ```shell
+        kubectl get pods -o wide
+    ```
+
+	- If all the pods are operating correctly, verify their connection with the Palette API. 
+		- For Clusters using Gateway, verify the connection between the Installer and Gateway instance:
+        ```shell
+           curl -k https://<KUBE_API_SERVER_IP>:6443
+        ```
+		- For Public Clouds that do not use Gateway, verify the connection between the public Internet and the Kube endpoint:
+        ```shell
+            curl -k https://<KUBE_API_SERVER_IP>:6443
+        ```
+
+        <InfoBox>
+            You can obtain the URL for the Kubernetes API using this command: ``kubectl cluster-info`` 
+        </InfoBox>
+
+4. If the Kubelet service does not work as expected, do the following. If the service operates correctly, you can skip this step. 
```

Comment:
```suggestion
4. If the kubelet service does not work as expected, do the following. If the service operates correctly, you can skip this step. 
```

Reply:
@ritawatson this is the name of the agent, so it should be capitalized.